.github/workflows/main.yml

```diff
@@ -23,7 +23,8 @@ jobs:
          texlive \
          texlive-latex-extra
         python -m pip install --upgrade pip
-        pip install matplotlib numpy scipy sympy pycddlib cvxpy
+        python -m pip install -r requirements.txt
```

Comment:
we already have sphinx/pip_requirements.txt.

I'd suggest that we remove that file, rename the new requirements.txt to pip_requirements.txt, and change the line in test.sh from
`$PYTHON -m pip freeze | grep -f sphinx/pip_requirements.txt`
to
`$PYTHON -m pip freeze | grep -f ./pip_requirements.txt`